pkgs/os-specific/bsd/netbsd/default.nix

```
@@ -651,7 +651,9 @@ let
     path = "lib/libm";
     version = "8.0";
     sha256 = "0i22603cgj6n00gn2m446v4kn1pk109qs1g6ylrslmihfmiy2h1d";
+    SHLIBINSTALLDIR = "$(out)/lib";
     meta.platforms = lib.platforms.netbsd;
+    extraPaths = [ sys.src ];
```

This will need
```suggestion
    extraPaths = with self; [ sys.src ];
```
I'm afraid